inbm/dispatcher-agent/dispatcher/source/ubuntu_source_manager.py

```diff
@@ -146,26 +146,26 @@ def remove(self, parameters: ApplicationRemoveSourceParameters) -> None:
         # Remove the file under /etc/apt/sources.list.d
         try:
             if (
-                os.path.sep in parameters.file_name
-                or parameters.file_name == ".."
-                or parameters.file_name == "."
+                os.path.sep in parameters.source_list_file_name
+                or parameters.source_list_file_name == ".."
+                or parameters.source_list_file_name == "."
             ):
-                raise SourceError(f"Invalid file name: {parameters.file_name}")
+                raise SourceError(f"Invalid file name: {parameters.source_list_file_name}")
```

Comment:
**Suggestion:** When raising exceptions, include the invalid value within quotes to clearly indicate the problematic input. This makes debugging easier by highlighting the exact value causing the issue. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
                raise SourceError(f"Invalid file name: '{parameters.source_list_file_name}'")
```